src/clj_github/httpkit_client.clj

```diff
@@ -11,14 +11,19 @@
 (defn get-installation-token [{:keys [token-fn]}]
   (token-fn))
 
+(defn- append-url-path [baseurl path]
+  (str baseurl (when-not (or (.endsWith baseurl "/")
+                             (.startsWith path "/"))
+                 "/") path))
+
 (defn- prepare
-  [{:keys [token-fn]} {:keys [path method body] :or {method :get} :as request}]
+  [{:keys [token-fn]} {:keys [path method body] :or {method :get path ""} :as request}]
   (-> request
       (assoc :method method)
       (assoc-some :body (and body (cheshire/generate-string body)))
-      (assoc :url (str github-url path))
-      (assoc-in [:headers "Content-Type"] "application/json")
-      (assoc-in [:headers "Authorization"] (str "Bearer " (token-fn)))))
+      (assoc :url (append-url-path github-url path))
+      (update-in [:headers "Content-Type"] #(or % "application/json"))
```

Comment:
what do you think about using `(fnil identity "application/json")`

Reply:
It's equal I guess, however I think `#(or % :foo)` is easier to understand.